src/node.js

```diff
@@ -73,6 +76,30 @@
         NativeModule.require('_third_party_main');
       });
 
+    } else if (process.openManPage) {
+      // Attempt to spawn `man 1 node`.
+      const cp = NativeModule.require('child_process');
+      const fs = NativeModule.require('fs');
+      const os = NativeModule.require('os');
+      const path = NativeModule.require('path');
+
+      const manual_path = path.join(os.tmpdir(), 'node.1');
+
+      try {
+        fs.writeFileSync(manual_path, manual);
+
+        const out = cp.spawnSync('man', [manual_path], { stdio: 'inherit' });
+        if (out.status === 0)
+          return process.exit(0);
+      } catch (er) {
+        console.error(`(${process.release.name}:${process.pid}) ${er.message}`);
+      } finally {
+        // Fall back to `node --raw-help`.
+        const out = cp.spawnSync(process.execPath, ['--raw-help'],
+                           { stdio: 'inherit' });
+        process.exit(out.status);
```

Comment:
I'd add a binding function for PrintHelp().  Trying to exec ourselves again has too many failure modes.